src/main/java/net/glowstone/io/entity/LivingEntityStore.java

```diff
@@ -148,6 +130,68 @@ public void load(T entity, CompoundTag compound) {
         }
     }
 
+    private void loadEquipment(T entity, EntityEquipment equip, CompoundTag compound) {
+        // Deprecated since 15w31a, left here for compatibilty for now
+        if (compound.isList("Equipment", TagType.COMPOUND)) {
+            List<CompoundTag> list = compound.getCompoundList("Equipment");
+            if (list.size() == 5) {
```

Comment:
Would it be better to allow for a variable size of this list? For example, having a list with only 1 element would only set the "item in hand" slot, and so forth.